### PR TITLE
fix: using 'vimdoc' as vim ft compiler was a bit aggressive

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
// trying to mdf if really to keep it as a 'compiler' 
// or maybe just back out #14459

### CODE REVIEW

**Comment by Freed-Wu** [comment]
Maybe we can use vint as default compiler of vim https://github.com/Vimjas/vint/issues/393

File: runtime/compiler/vimdoc.vim [hunk 1]
```diff
@@ -1,9 +1,10 @@
 " Vim Compiler File
 " Language:             vimdoc
 " Maintainer:           Wu, Zhenyu <wuzhenyu@ustc.edu>
-" Latest Revision:      2024-04-09
+" Latest Revision:      2024-04-13
 "
-" you can get it by `pip install vimdoc` or the package manager of your distribution.
+" if can not find 'vimdoc' in the package manager of your distribution e.g 'pip',
```
Comment (zeertzjq) [inline runtime/compiler/vimdoc.vim]:
```suggestion
" If you cannot find 'vimdoc' in the package manager of your distribution e.g 'pip',
```

File: runtime/compiler/vimdoc.vim [hunk 2]
```diff
@@ -13,7 +14,7 @@ let b:current_compiler = 'vimdoc'
 let s:save_cpoptions = &cpoptions
 set cpoptions&vim
 
-CompilerSet makeprg=vimdoc
+CompilerSet makeprg=vimdoc\ .
```
Comment (chrisbra) [inline runtime/compiler/vimdoc.vim]: Please remove the dot.

**Comment by Shane-XB-Qian** [comment]
> -CompilerSet makeprg=vimdoc
> +CompilerSet makeprg=vimdoc\ .
> 
> Please remove the dot.

ah.. remove this `dot` maybe worse, it probably would raise err or err msg,
but i would not insist it, mostly just was for making it not impact vim ft,
so if you confirm (and not to back out) i will remove that `dot`.

-- 
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
> -CompilerSet makeprg=vimdoc
> +CompilerSet makeprg=vimdoc\ .
> 
> Please remove the dot.

nevermind, i felt removed that `dot` should be a bit safer than had, since seems it would override `./doc` by force.

-- 
shane.xb.qian

**Comment by chrisbra** [comment]
Thanks.

**Pull request merged**